src/org/parosproxy/paros/model/SiteMap.java

```diff
@@ -834,6 +834,10 @@ private void handleEvent(SiteNode parent, SiteNode node, EventType eventType) {
 	private static void publishEvent(String event, SiteNode node) {
 		ZAP.getEventBus().publishSyncEvent(SiteMapEventPublisher.getPublisher(), new Event(SiteMapEventPublisher.getPublisher(), event, new Target(node)));
 	}
+
+	public SiteNode getRootNode() {

```

Comment:
It could override `getRoot()` instead (`getRoot()` is already in use in other places, with an explicit cast).